atom/lib/import-js.js

```diff
@@ -0,0 +1,302 @@
+'use strict';
+
+const AskForSelectionView = require('./AskForSelectionView');
```

Comment:
I think I did, but since I didn't have it configured it gave me non-relative paths that were all wrong, so I had to do some manual stuff. It would be nice if we finished #186 so that things would work better out of the box.